### PR TITLE
Add Netheopoiesis

### PR DESCRIPTION
**Pull request by Sefiraat**
Builds page link will not function until the builds PR has been approved.

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ Missile Warfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare
...
+Netheopoiesis | Sefiraat / J3fftw | <details><summary>expand</summary>This addon was our entry into the Slimefun Addon Jam 2022. The premise of the addon is to allow players to play vanilla Minecraft and Slimefun solely within the Nether. You can purify and beautify the nether over time including changing biomes, spawns and lots lots more. This is held up by a robust plant growing/breeding and harvesting system with plants that fullfill many functions including active like spawnings, growing trees, dropping items and more. A branching but linear progression through the addon should provide lots of content, with more to come.</details> | [Readme](https://github.com/Sefiraat/Netheopoiesis/blob/master/README.md) | [Github](https://github.com/Sefiraat/Netheopoiesis) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Netheopoiesis/master/)
```
Comment (svr333) [inline pages/Addons.md:76]: I assume at least thats what it was supposed to be
also "spawnings" isn't a word afaik, not sure what you meant there though
```suggestion
Netheopoiesis | Sefiraat / J3fftw | <details><summary>expand</summary>This addon was our entry into the Slimefun Addon Jam 2022. The premise of the addon is to allow players to play vanilla Minecraft and Slimefun solely within the Nether. You can purify and beautify the nether over time including changing biomes, spawns and lots lots more. This is held up by a robust plant growing/breeding and harvesting system with plants that fullfill many functions including activities like spawnings, growing trees, dropping items and more. A branching but linear progression through the addon should provide lots of content, with more to come.</details> | [Readme](https://github.com/Sefiraat/Netheopoiesis/blob/master/README.md) | [Github](https://github.com/Sefiraat/Netheopoiesis) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Netheopoiesis/master/)
```

**Review by svr333 — CHANGES_REQUESTED** [review]

File: pages/Addons.md (hunk 1 above)
Comment (Sefiraat) [reply pages/Addons.md:76]: How very dare you :D

Comment (svr333) [reply pages/Addons.md]: get englished

**Review by svr333 — DISMISSED** [review]

**Review by Phoenix-Starlight — APPROVED** [review]
I assume by 'spawnings' you mean 'spawning mobs'. Otherwise its good to go!

**Comment by Sefiraat** [comment]
I really should be better with my own english skills ha, ty @Phoenix-Starlight ;)

**Comment by svr333** [comment]
Dont worry youre only a native english speaker :)

**Review by svr333 — APPROVED** [review]

**Pull request merged**